File: src/main/java/dev/slne/surf/essentials/commands/cheat/gui/LoomCommand.java
package dev.slne.surf.essentials.commands.cheat.gui;

import dev.slne.surf.essentials.utils.permission.Permissions;

public class LoomCommand extends GuiCommand {
    LoomCommand() {
        super("loom", "Der Webstuhl", Permissions.LOOM_SELF_PERMISSION, Permissions.LOOM_OTHER_PERMISSION);
    }

    @Override
    public void open(org.bukkit.entity.Player player) {
        player.openLoom(player.getLocation(), true);
    }
}

File: src/main/kotlin/dev/slne/surf/essentials/command/table/LoomCommand.kt
@file:Suppress("DEPRECATION")

package dev.slne.surf.essentials.command.table

import dev.jorel.commandapi.kotlindsl.commandTree
import dev.jorel.commandapi.kotlindsl.playerExecutor
import dev.slne.surf.essentials.util.permission.EssentialsPermissionRegistry
import dev.slne.surf.surfapi.core.api.messages.adventure.sendText

fun loomCommand() = commandTree("loom") {
    withPermission(EssentialsPermissionRegistry.LOOM_COMMAND)
    playerExecutor { player, _ ->
        player.openLoom(null, true)
        player.sendText {
            appendPrefix()
            success("Du hast einen Webstuhl geöffnet.")
        }
    }
}